wp-includes/widgets/class-wp-widget-image.php

```diff
@@ -33,6 +33,10 @@ public function __construct() {
 			'edit_media' => __( 'Edit Image' ),
 			'change_media' => __( 'Change Image' ),
 			'select_media' => __( 'Select Image' ),
+			'error' => sprintf(
+				__( 'We can&#8217;t find that image. Check your <a href="%s">media library</a> and make sure it wasn&#8217;t deleted.' ),
```

Comment:
The Backbone view for should have a new event handler for clicking on this link to have the same result as clicking the "Select Media" or "Change Media" buttons. The underlying `href` can remain there so they can right-click to open the media library in a new tab, but otherwise we should keep them inside the customizer and open the select frame for the media modal.